app.js

```diff
@@ -1,3 +1,33 @@
+var express = require('express');
+var trucks = require('./trucks');
+
+var app = express();
+
+app.set('port', 3000);
+app.use(express.static('public'));
+
+app.get('/trucks', function(request, response) {
+	var allTrucks = trucks.getTrucks();
+	response.send(allTrucks);
+});
+
+app.get('/trucks/:trucks', function(request,response) {
+	var truckName = request.params.trucks;
+	response.send(trucks.getTruck(truckName)[0]);
+});
+
+app.get('/food-types', function(request, response) {
+	var foodTypes = {
+		foodTypes: trucks.getFoodTypes()
+	}
+	response.send(foodTypes);
```

Comment:
Like the getTrucks method, I would return the result of getFoodTypes() directly as it's easier for the caller to consume the data that way.